generators/app/files.js

```diff
@@ -88,11 +88,18 @@ module.exports.CUSTOM_TEXT_STYLES = `${module.exports.CUSTOM_TEXT_PATH}/styles.j
 
 // COMPONENTS - CUSTOM TEXT INPUT
 module.exports.CUSTOM_TEXT_INPUT_PATH = `${module.exports.COMPONENTS_PATH}/CustomTextInput`;
-module.exports.CUSTOM_TEXT_INPUT = `${module.exports.CUSTOM_TEXT_INPUT_PATH}/index.js`;
-module.exports.CUSTOM_TEXT_INPUT_STYLES = `${module.exports.CUSTOM_TEXT_INPUT_PATH}/styles.js`;
+module.exports.CUSTOM_TEXT_INPUT = `${module.exports.CUSTOM_TEXT_INPUT_PATH}/index.tsx`;
+module.exports.CUSTOM_TEXT_INPUT_STYLES = `${module.exports.CUSTOM_TEXT_INPUT_PATH}/styles.ts`;
+module.exports.CUSTOM_TEXT_INPUT_INTERFACE = `${module.exports.CUSTOM_TEXT_INPUT_PATH}/interface.ts`;
 module.exports.CUSTOM_TEXT_INPUT_ASSETS_PATH = `${module.exports.CUSTOM_TEXT_INPUT_PATH}/assets`;
 module.exports.CTI_SHOW_PASSWORD_PATH = `${module.exports.CUSTOM_TEXT_INPUT_PATH}/components/ShowPassword`;
 
+// COMPONENTS - ANIMATED CUSTOM TEXT INPUT
+module.exports.ANIMATED_CUSTOM_TEXT_INPUT_PATH = `${module.exports.CUSTOM_TEXT_INPUT_PATH}/AnimatedCustomTextInput`;
```

Comment:
also, use only `ANIMATED_CUSTOM_TEXT_INPUT_PATH`, the other ones are not needed, the boostrap copies the whole folder, check it. Same with `CUSTOM_TEXT_INPUT_PATH`

Reply:
They are in the same folder because both components have other component and some assets in common. It's something that we decided before. but what do you think is better?